specification/datalake-analytics/data-plane/readme.nodejs.md

```diff
@@ -24,4 +25,7 @@ nodejs:
 ``` yaml $(nodejs) && $(tag) == 'package-job'
 nodejs:
   source-code-folder-path: lib/job
-```
\ No newline at end of file
+  generate-license-txt: true
```

Comment:
Revert the changes in this file. I just made these changes and this PR is attempting to undo them.

Reply:
I think I messed up a merge.  Please stand by...